src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -59,6 +79,65 @@ public void AppendParts(IEnumerable<TaggedText> parts)
                 _anyNonWhitespaceSinceLastPara = true;
             }
 
+            public void PushList(DocumentationCommentListType listType)
+            {
+                _listStack.Add((listType, 0, false));
+                MarkBeginOrEndPara();
+            }
+
+            public void NextListItem()
+            {
+                if (_listStack.Count == 0)
+                {
+                    return;
+                }
+
+                var (type, index, renderedItem) = _listStack[_listStack.Count - 1];
+                if (renderedItem)
+                {
+                    Builder.Add(new TaggedText(TextTags.ContainerEnd, string.Empty));
+                }
+
+                _listStack[_listStack.Count - 1] = (type, index + 1, false);
+                MarkLineBreak();
+            }
+
+            public void PopList()
```

Comment:
going to skip to the end for now.  woudl like docs so i can wrap my head around things and validate it.  thanks!